composer.json

```diff
@@ -25,10 +25,10 @@
             "FriendsOfCake\\TestUtilities\\Test\\": "tests"
         }
     },
-    "require": {
-        "php": ">=5.4"
-    },
     "require-dev": {
-        "phpunit/phpunit": "~4.0"
+        "phpunit/phpunit": "^6.0"
+    },
+    "require": {
+        "cakephp/cakephp": "^3.5"
```

Comment:
I would usually keep the PHP min req for clarity and for travis matrix prefer min possibilities.

Reply:
Yeah we be nice if we could keep this to `3.4` as that's the minimum version most FOC plugins use.